Platform/ElkhartlakeBoardPkg/Library/Stage1BBoardInitLib/Stage1BBoardInitLib.c

```diff
@@ -875,6 +875,8 @@ RtcInit (
   Bar     = MmioRead32 (MM_PCI_ADDRESS (0, PCI_DEVICE_NUMBER_PCH_PMC, PCI_FUNCTION_NUMBER_PCH_PMC, R_PMC_CFG_BASE)) & ~0x0F;
   PmConf1 = MmioRead8 (Bar + R_PMC_PWRM_GEN_PMCON_B);
 
+  RtcRead (R_RTC_IO_REGA);
```

Comment:
Is this code belongs to other patch?

Reply:
For this patch

Reply:
I mean that can be put to separate commit with commit message for the change, but as is also OK.